tests/qhub-config-yaml-files-for-upgrade/qhub-config-do-310-customauth.yaml

```diff
@@ -22,23 +22,23 @@ security:
     admin:
       gid: 101
 default_images:
-  jupyterhub: quansight/qhub-jupyterhub:v0.3.10
-  jupyterlab: quansight/qhub-jupyterlab:v0.3.10
-  dask_worker: quansight/qhub-dask-worker:v0.3.10
-  dask_gateway: quansight/qhub-dask-gateway:v0.3.10
+  jupyterhub: quansight/nebari-jupyterhub:v0.3.10
+  jupyterlab: quansight/nebari-jupyterlab:v0.3.10
+  dask_worker: quansight/nebari-dask-worker:v0.3.10
+  dask_gateway: quansight/nebari-dask-gateway:v0.3.10
 storage:
   conda_store: 60Gi
   shared_filesystem: 100Gi
 theme:
   jupyterhub:
-    hub_title: QHub - do-pytest
+    hub_title: Nebari - do-pytest
     hub_subtitle: Autoscaling Compute Environment on Digital Ocean
-    welcome: Welcome to do.qhub.dev. It is maintained by <a href="http://quansight.com">Quansight
+    welcome: Welcome to do.nebari.dev. It is maintained by <a href="http://quansight.com">Quansight
       staff</a>. The hub's configuration is stored in a github repository based on
-      <a href="https://github.com/Quansight/qhub/">https://github.com/Quansight/qhub/</a>.
-      To provide feedback and report any technical problems, please use the <a href="https://github.com/Quansight/qhub/issues">github
+      <a href="https://github.com/Quansight/nebari/">https://github.com/Quansight/nebari/</a>.
+      To provide feedback and report any technical problems, please use the <a href="https://github.com/Quansight/nebari/issues">github
```

Comment:
```suggestion
      To provide feedback and report any technical problems, please use the <a href="https://github.com/nebari-dev/nebari/issues">github
```

Reply:
need to make sure this is fixes in all the ocurrences

Reply:
See issue #1512